app/build.gradle

```diff
@@ -30,8 +34,12 @@ dependencies {
     implementation 'com.android.support:design:26.1.0'
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     implementation 'com.android.support:support-vector-drawable:26.1.0'
+    implementation 'com.github.arimorty:floatingsearchview:2.1.1'
+    implementation 'com.github.cuappdev:futurenovajava:ba31faf'
```

Comment:
when you use the new lib you need to use the new commit. Eventually we will use the release feature on github

Reply:
Updated to version 002463b.